packages/block-library/src/separator/editor.scss

```diff
@@ -2,11 +2,4 @@
 	// Prevent margin collapsing so the area to select the separator is bigger.
 	padding-top: 0.1px;
 	padding-bottom: 0.1px;
-
-	// This is also set in style.scss, but needs a higher specificity in editor
-	// due to the way that color block supports adds additional background color styles.
-	&.wp-block-separator.is-style-dots {
-		background: none !important;
-		border: none;
-	}
```

Comment:
This removal appears to break setting a background color on an individual block in the post editor:

<img width="908" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/95926262-cc50-4202-97d4-12b9e5be7972">

It's fine on the site frontend, but not in the editor.